src/Bicep.Core/Syntax/SyntaxFactory.cs

```diff
@@ -113,5 +144,42 @@ private static string EscapeBicepString(string value)
             .Replace("\t", "\\t")
             .Replace("${", "\\${")
             .Replace("'", "\\'");
+
+        private static string TryGetTokenText(TokenType tokenType) => tokenType switch

```

Comment:
>TryGetTokenText [](start = 30, length = 15)

I'm fine with it as-is, but don't we have a similar list somewhere else?

Reply:
There's a list for binary operators conversion which is somewhat similar, but we don't have one for converting all token types.